.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@blumilksoftware/blumilk
```

Comment:
```suggestion
@blumilksoftware/interns2024b
```

Reply:
Or maybe better - instead of this, please add me and @kamilpiech97 as code owners